package.json

```diff
@@ -160,6 +160,7 @@
   },
   "dependencies": {
     "@folio/react-intl-safe-html": "^1.0.2",
+    "@folio/stripes-util": "^2.0.0",
```

Comment:
Please import this via `@folio/stripes/util` instead.

Reply:
i.e. we don't need a direct dependency because we can rely on the `@folio/stripes` peerDependency.

Reply:
@zburke, I don't know why, but when importing via '@folio/stripes/util' the 'copyNumber' value doesn't appear in the 'Effective call number string' field.

Reply:
What the ...? Makes me wonder if the correct version of stripes-util is being wrapped into stripes.